src/teamsnap-ui.scss

```diff
@@ -1,3 +1,5 @@
+@import "config/index";
 @import "base/index";
+@import "fonts/index";
```

Comment:
Done, I agree that things with no dependencies are better off being imported first.